jenkins-job.sh

```diff
@@ -606,7 +606,7 @@ function run_halium {
     [[ -d ${BUILD_DIR} ]] || mkdir ${BUILD_DIR}
     cd ${BUILD_DIR}
 
-    rm -rf .repo/local_manifests/
+    rm -rf .repo/local_manifests/ | rm -rf bionic | rm -rf device | rm -rf kernel | rm -rf vendor
```

Comment:
no need for chaining, just do rm -rf .repo/local_manifests/ bionic device kernel vendor